met-web/src/services/commentService/index.tsx

```diff
@@ -29,13 +29,14 @@ export const getComment = async (commentId: number): Promise<Comment> => {
     }
 };
 
-interface UpdateCommentRequest {
-    id: string;
-    survey_id: number;
+interface ReviewCommentRequest {
+    status_id: number;
+    comment_id: number;
 }
-export const UpdateComment = async (data: UpdateCommentRequest): Promise<Comment> => {
+export const ReviewComment = async ({ comment_id, status_id }: ReviewCommentRequest): Promise<Comment> => {
```

Comment:
in the frontend we use camelCase commentId and the backend the snake case comment_id, quite confusing, maybe lets have a task to standardize

Reply:
problem is camelCase is kinda the standard in JS and snake case in python. Choosing one for both maybe is choosing ones standard over the other. Could be wrong.